packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -151,10 +153,43 @@ export class BlockList extends Component {
 			isReadOnly,
 			shouldShowInsertionPointBefore,
 			shouldShowInsertionPointAfter,
+			containerStyle,
+			getBlockAttributes,
 		} = this.props;
 
+		const getVerticalAlignmentRemap = ( newAlignment ) => {
```

Comment:
on web this attribute is 'top' / 'center' / 'bottom' 

I need to remap to flex props name